subgraphs/curve-finance/src/common/constants.ts

```diff
@@ -95,6 +95,10 @@ export namespace NULL {
 export namespace Protocol {
```

Comment:
Doesn’t this file need to be deleted entirely?

Reply:
yeah I will delete this file.